elife/php7.sls

```diff
@@ -91,7 +91,7 @@ install-composer:
         - require:
             - cmd: php
             - environ: composer-home
-            - environ: composer-auth
+            - composer-auth
```

Comment:
I trust you, I just can't find mention of this change in the release notes ...
https://docs.saltstack.com/en/latest/topics/releases/2016.3.0.html